cmd/controller-manager/app/controller-manager.go

```diff
@@ -22,6 +22,7 @@ import (
 	"flag"
 	"fmt"
 	"io/ioutil"
+	"k8s.io/klog"
```

Comment:
duplicate import?
```go
import (
    ...
    "k8s.io/klog/v2"
)
```

Reply:
thanks for pointing out

Reply:
https://github.com/kubernetes-sigs/kubefed/pull/1366#issuecomment-787560939